### PR TITLE
Translate 2024 survey into zh-tw

### PR DESCRIPTION
**Pull request by lili668668**

### CODE REVIEW

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 1]
```diff
@@ -0,0 +1,909 @@
...
+
+### 您在使用 Rust 時遇到了哪些挑戰？
+
+Type: free form
+
+### 貴公司在哪些技術領域使用 Rust？
+
+Type: select all that apply
+
+- 音檔編程
```
Comment (weihanglo) [inline surveys/2024-annual-survey/translations/zh-tw.md:526]: 音訊程式設計

Comment (weihanglo) [inline surveys/2024-annual-survey/translations/zh-tw.md:526]:
```suggestion
- 音訊程式設計
```

**Review by weihanglo — APPROVED** [review]
Thanks a lot!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 2]
```diff
@@ -0,0 +1,909 @@
...
+- 個人教學
+- 大學教學資料
+- 其他（請注明）
+
+### 您在今年或去年是否參加過（或仍在參加）使用或教授 Rust 的課程或培訓？
+
+Type: select one
+
+- 是，通過大學、學校或其他教育機構
+- 是，通過我的雇主、承包商或諮詢公司
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:95]:
```suggestion
- 是，透過大學、學校或其他教育機構
- 是，透過我的雇主、承包商或諮詢公司
```

Maybe this? 🤔

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 3]
```diff
@@ -0,0 +1,909 @@
...
+
+Type: select all that apply
+
+- 我不使用 Nightly
+- 出於習慣
+- 為了我需要的特定語言功能
+- 我喜歡能用上所有最新功能
+- 幫助測試；檢測 Nightly 版本是否含有缺陷
+- 提供有關 Nightly 功能的設計反饋
+- 用於持續集成中測試
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:167]: Maybe we won't translate CI like this? 🤔 

```suggestion
- 用於 CI 中測試
```

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 4]
```diff
@@ -0,0 +1,909 @@
...
+Columns:
+
+- 同意
+- 不同意
+
+### 請問您對於 Rust 的發展有何評價？
+
+Type: select one
+
+- 我認為 Rust 已經過於覆雜，不應添加新的顯著功能或使更多僅 Nightly 可用的顯著功能穩定化
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:285]:
```suggestion
- 我認為 Rust 已經過於複雜，不應添加新的顯著功能或使更多僅 Nightly 可用的顯著功能穩定化
```

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 5]
```diff
@@ -0,0 +1,909 @@
...
+- 制作關於 Rust 的訊息內容（如部落格、直播、制作 YouTube 影片、在會議/聚會上展示等
+- 接收有關 Rust 的訊息內容（如部落格、直播、YouTube 影片等）
+- 閱讀 Rust 官方溝通渠道（如 This Week in Rust、Rust 官方部落格、Rust 推特帳號等）
+- 參與社交媒體或網站（Hacker News、r/rust、Twitter、LinkedIn等）上關於Rust的討論
+- 參加 Rust 聚會或會議（虛擬或面對面）
+- 撰寫、評論、參與討論或編輯公開的 RFC
+- 在官方聊天室（internals.rust-lang.org, rust-lang Zulip 或 Discord）中討論 Rust 專案
+- 在 rust-lang GitHub 組織中的任何倉庫報告問題
+- 為 rust-lang GitHub 組織里的任何專案貢獻程式碼變更（包括測試）
+- 為 rust-lang GitHub 組織里的任何專案貢獻非程式碼的變更（如文件、注釋等）
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:427]:
```suggestion
- 在 rust-lang GitHub 組織中的任何專案報告問題
- 為 rust-lang GitHub 組織裡的任何專案貢獻程式碼變更（包括測試）
- 為 rust-lang GitHub 組織裡的任何專案貢獻非程式碼的變更（如文件、註解等）
```

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 6]
```diff
@@ -0,0 +1,909 @@
...
+### 以下哪些描述是您在工作中使用 Rust 的原因？
+
+Type: select all that apply
+
+- 性能（即速度、記憶體用量等）特征
+- 我們需要精確控制軟體的運行行為
+- Rust 的安全性和可靠性對我們來說非常重要
+- Rust 允許我們構建相對正確和無缺陷的軟體
+- 我們發現使用 Rust 編程是令人愉快或有趣的
+- 我們已經知道 Rust，所以它是我們的默認選擇
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:514]:
```suggestion
- 我們已經知道 Rust，所以它是我們的預設選擇
```

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 7]
```diff
@@ -0,0 +1,909 @@
...
+
+- 在業界使用不足
+- 太多來自大公司的興趣
+- 對生態的開源貢獻不足
+- 未加入我想要的特定語言功能
+- Rust 的發展不夠快
+- 語言不穩定
+- 被替代品取代
+- 變得過於覆雜
+- 工具和文件未能提供足夠的無障礙支持（如自然語言的障礙或不兼容屏幕閱讀器）
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:616]:
```suggestion
- 工具和文件未能提供足夠的無障礙支援（如自然語言的障礙或不相容螢幕閱讀器）
```

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 8]
```diff
@@ -0,0 +1,909 @@
...
+
+### 您是全日制學生或者非全日制學生嗎？
+
+Type: select one
+
+- 不是
+- 是，目前是初中生/高中生
+- 是，目前是本科生
+- 是, 目前是碩士生
+- 是, 目前是博士生
```
Comment (CYBAI) [inline surveys/2024-annual-survey/translations/zh-tw.md:658]:
```suggestion
- 是，目前是碩士生
- 是，目前是博士生
```

**Review by CYBAI — APPROVED** [review]
THANK YOU SO MUCH!! 🙏 ❤️ 🙏

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 9]
```diff
@@ -0,0 +1,909 @@
...
+- 2022年或2023年期間
+- 2020年或2021年期間
+- 2018年或2019年期間
+- 2017年或之前
+
+### 您從哪裡學習 Rust ？
+
+Type: select all that apply
+
+- 書籍（例如《The Rust 編程語言》、《Rust for Rustaceans》等）
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 10]
```diff
@@ -0,0 +1,909 @@
...
+- 線上練習教學（例如 Rustlings、Rust by Examples 等）
+- 影片或直播
+- 部落格
+- 官方文件
+- crate 的原始碼
+- 線上課程、網路研討會
+- 個人教學
+- 大學教學資料
+- 其他（請注明）
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 11]
```diff
@@ -0,0 +1,909 @@
...
+- 特化 (Specialization)
+- 生成器/協程 (Generators/coroutines)
+- 異步閉包 (Async closures)
+- 異步生成器/協程 (Async generators/coroutines)
+- if/while let鏈 (If/while let chains)
+- try 表達式塊 (Try blocks)
+- 永無(never)類型 (Never type)
+- 特質別名 (Trait aliases)
+- 類型別名中的 impl Trait (Type Alias Impl Trait (TAIT) )
+- 默認關聯類型 (Associated type defaults)
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 12]
```diff
@@ -0,0 +1,909 @@
...
+### 您希望哪些目前處於未實現狀態（或僅 Nightly 可用）的功能進行穩定化？
+
+Type: matrix
+
+Rows:
+
+- 特化 (Specialization)
+- 生成器/協程 (Generators/coroutines)
+- 異步閉包 (Async closures)
+- 異步生成器/協程 (Async generators/coroutines)
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 13]
```diff
@@ -0,0 +1,909 @@
...
+- 跨性別
+- 女性或被視為女性
+- 其他
+
+### 您是全日制學生或者非全日制學生嗎？
+
+Type: select one
+
+- 不是
+- 是，目前是初中生/高中生
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 14]
```diff
@@ -0,0 +1,909 @@
...
+- 是，目前是本科生
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 15]
```diff
@@ -0,0 +1,909 @@
...
+Type: select all that apply
+
+- 在業界使用不足
+- 太多來自大公司的興趣
+- 對生態的開源貢獻不足
+- 未加入我想要的特定語言功能
+- Rust 的發展不夠快
+- 語言不穩定
+- 被替代品取代
+- 變得過於覆雜
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 16]
```diff
@@ -0,0 +1,909 @@
...
+- 同意
+- 不同意也不反對
+- 反對
+
+### 您覺得以下對 Rust 的描述哪些是符合實際情況的？
+
+Type: select all that apply
+
+- Rust 比其他程式語言提供了真正的優勢
+- Rust 的編程比其他程式語言覆雜得多
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 1 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 2 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 3 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 4 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 5 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 7 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 8 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md (hunk 6 above)
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

File: surveys/2024-annual-survey/translations/zh-tw.md [hunk 17]
```diff
@@ -0,0 +1,909 @@
...
+### 如果您使用 Nightly 版，為什麽？
+
+Type: select all that apply
+
+- 我不使用 Nightly
+- 出於習慣
+- 為了我需要的特定語言功能
+- 我喜歡能用上所有最新功能
+- 幫助測試；檢測 Nightly 版本是否含有缺陷
+- 提供有關 Nightly 功能的設計反饋
```
[the remark this answers is not in the document]
Comment (lili668668) [reply surveys/2024-annual-survey/translations/zh-tw.md]: Thanks!

**Review by weihanglo — APPROVED** [review]
@Kobzol

This looks good enough. Maybe we can merge and reuse in 2025?

**Comment by Kobzol** [comment]
Yeah :) Thank you!

**Pull request merged**